docs/api.rst

```diff
@@ -139,83 +139,69 @@ Variable recombination rates
     :members:
 
 
-*******************
-Processing results
-*******************
+************************
+Using simulation results
+************************
 
 The :class:`.TreeSequence` class represents a sequence of correlated trees
 output by a simulation. The :class:`.SparseTree` class represents a single
-tree in this sequence.
+tree in this sequence. 
+These classes are the interfaces used to interact with the trees
+and mutational information stored in a tree sequence returned from a simulation.
+There are also methods for loading data into these objects, either from the native
+format, e.g., :func:`msprime.load`, or from another format, e.g.,
+:func:`msprime.load_text` compatible with the `Tables API <sec_tables_api>`.
+
++++++++++++++++++
+Top level-classes
++++++++++++++++++
+
+
+.. autoclass:: msprime.TreeSequence()
+    :members:
+
+.. autoclass:: msprime.SparseTree()
+    :members:
+
 
 +++++++++
 Constants
 +++++++++
 
-.. data:: msprime.NULL_NODE = -1
+These constants are used in :class:`.TreeSequence` and :class:`.SparseTree`
+instances to define special values of various variables.
+
+.. data:: msprime.NULL_NODE == -1
```

Comment:
What's the difference with == here?

Reply:
I changed to == because the single = looked like assignment, to me. With equality I can paste it into an interpreter and see 

```
In [4]: msprime.NULL_NODE == 1
Out[4]: True
```
(Of course, I could paste single = in too, but we want to discourage changing these manually in an imported module ;)

Happy to change back if it reads weirdly.